Parse/Internal/File/NikitaInputStream/NikitaInputStream.m

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright (c) 2015-present, Parse, LLC.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import "NikitaInputStream.h"
+
+#import <objc/runtime.h>
+#import <objc/message.h>
+
+@interface NikitaInputStream() <NSStreamDelegate>
+@end
+
+@implementation NikitaInputStream {
+    NSString *_path;
```

Comment:
Nit: Move all instance variables into class extension above to be consistent in code style.